packages/contracts-bedrock/test/kontrol/scripts/run-kontrol.sh

```diff
@@ -94,14 +94,14 @@ kontrol_prove() {
                         --max-depth ${max_depth}           \
                         --max-iterations ${max_iterations} \
                         --smt-timeout ${smt_timeout}       \
-                        --bmc-depth ${bmc_depth}           \
```

Comment:
Why is this no longer needed?

Reply:
We're not performing bounded model checking for this project. I was experimenting at some point with this option and forgot to delete it. It did not affect the `SetNumber` dummy proof we were running, but we want to turn it off for running the actual proofs. The reason we don't need bounded model checking is that we can reason in full generality in these proofs. Particularly, we don't need to restrict ourselves to unrolling the loops just a certain number of times